lib/js/components/buttons/Button/Button.consts.ts

```diff
@@ -10,17 +10,21 @@ export const BUTTON_TYPES = {
 	TEXT: 'text',
 } as const;
 
-export const BUTTON_COLORS = {
+export const BUTTON_ICON_COLORS = {
```

Comment:
IMO, it's easier to maintain when you join additional colors with `BUTTON_COLORS` instead of defining them twice